ipalib/constants.py

```diff
@@ -215,6 +215,7 @@
     ('script', object),  # sys.argv[0]
     ('bin', object),  # The directory containing the script
     ('home', object),  # $HOME
+    ('fips_mode', object),  # FIPS-mode of the server
```

Comment:
Won't this cause `api.env.fips_mode` to be `object` rather than unset when `ipaplatform` is not installed?